packages/alfa-css/src/properties/font-weight.ts

```diff
@@ -0,0 +1,31 @@
+import { parse } from "@siteimprove/alfa-lang";
+import { FontWeightGrammar } from "../grammars/font-weight";
+import { Property } from "../types";
+
+export type FontWeight = Readonly<
+  // https://www.w3.org/TR/css-fonts/#font-weight-absolute-values
+  | {
+      type: "absolute";
+      value: 100 | 200 | 300 | 400 | 500 | 600 | 700 | 800 | 900;
+    }
+  | {
+      type: "relative";
+      value: "bolder" | "lighter";
+    }
+>;
+
+/**
+ * @see https://www.w3.org/TR/css-fonts/#propdef-font-weight
+ */
+export const FontWeightProperty: Property<FontWeight> = {
+  inherits: true,
+  parse(input) {
+    return parse(input, FontWeightGrammar);
+  },
+  initial() {
+    return { type: "absolute", value: 400 };
+  },
+  computed(getProperty) {
+    return getProperty("fontWeight");
```

Comment:
Per the spec, the computed value of the `font-weight` property is a numeric weight value. The above will simply return the specified or inherited value, even if it's `bolder` or `lighter`, neither of which are numeric values. You will need to implement the `font-weight` step of the [font matching algorithm](https://www.w3.org/TR/css-fonts/#font-matching-algorithm) in order to determine to computed `font-weight` value. For now, we can assume that all weights are available regardless of `font-family`; we have yet to determine how to handle `font-family`, see https://github.com/auto-wcag/auto-wcag/issues/135.